src/WorkflowGui/Controller/WorkflowController.php

```diff
@@ -200,6 +200,11 @@ protected function sanitizeConfiguration($configuration)
     {
         if (isset($configuration['places'])) {
             foreach ($configuration['places'] as $placeKey => &$placeConfig) {
+                if (isset($placeConfig['color'])) {
+                    if (substr($placeConfig['color'], 0, 1) !== '#') {
+					    $placeConfig['color'] = '#'.$placeConfig['color'];
```

Comment:
still not good...

Reply:
Can I check with some tool and see what's not good? Thanks

Reply:
indentation is not good, $placeConfig is indented too much.

Reply:
Ah, damn tabs. Thanks